### PR TITLE
Added level class and option for auto indentation

### PR DESCRIPTION
**Pull request by eguitarz**
This PR contains two changes:
1) Added `level-${depth}` classname for each node row.
2) AsyncTree now accepts `autoIndentation` option to disable inline indentation.

### CODE REVIEW

**Comment by eguitarz** [comment]
@taras Would you mind to take a look of this?

File: app/templates/components/async-tree.hbs [hunk 1]
```diff
@@ -7,7 +7,7 @@
       {{#if node.isMore}}
         {{#async-more on-enter-viewport=(action 'fetchMore' meta)}}More{{/async-more}}
       {{else}}
-        <div class="node-list-item {{if node.isLoading 'is-loading'}} {{if (contains node openNodes) 'is-open' 'is-not-open'}} {{if node.isEmpty 'is-empty'}}" style={{node-style style node bind=this}}>
+        <div class="node-list-item level-{{node.depth}} {{if node.isLoading 'is-loading'}} {{if (contains node openNodes) 'is-open' 'is-not-open'}} {{if node.isEmpty 'is-empty'}}" style={{node-style style node autoIndent bind=this}}>
```
Comment (taras) [inline app/templates/components/async-tree.hbs]: What about doing something like this: 

```hbs
 <div class="node-list-item level-{{node.depth}} {{if node.isLoading 'is-loading'}} {{if (contains node openNodes) 'is-open' 'is-not-open'}} {{if node.isEmpty 'is-empty'}}" style={{if styleIndent (node-style style node bind=this}}>
```

Comment (eguitarz) [inline app/templates/components/async-tree.hbs]: cool, works for me, changing now

**Comment by eguitarz** [comment]
@taras Ready for your second review. Also I'll release it on npm once I merged this in.

**Comment by taras** [comment]
LGTM

**Pull request merged**